docs/docsite/rst/network/getting_started/network_differences.rst

```diff
@@ -0,0 +1,56 @@
+************************************************************
+How Network Automation is Different
```

Comment:
This is a very good explanation, thanks very much for including this as well!

Reply:
Glad you like it! I think this page is the most useful for the experienced Ansibler looking to bring network devices into the automation fold.